README.asciidoc

```diff
@@ -447,6 +447,50 @@ task functionalTestMyApp(type: Test, dependsOn: startMyAppContainer) {
 }
 ----
 
+== Interacting with a container running in the background
+
+This plugin can be used to launch and manage a background container to facilitate
+locally running a Dockerized application. This can be done by creating the container
+with an explicit `containerName` which is then used as the `targetContainerId` for
+relevant tasks.
+
+[source,groovy]
+----
+apply plugin: 'com.bmuschko.docker-remote-api'
+
+import com.bmuschko.gradle.docker.tasks.container.*
+
+ext {
+  CONTAINER_NAME = 'myapp'
+} 
+
+task buildMyAppImage(type: DockerBuildImage) {
+    inputDir = file('docker/myapp')
+    tag = 'test/myapp:latest'
+}
+
+task dockerCreateContainer(type: DockerCreateContainer) {
+    dependsOn buildMyAppImage
+    targetImageId { buildMyAppImage.imageId }
+    containerName = CONTAINER_NAME
+}
+
+task dockerStartContainer(type: DockerStartContainer) {
+    dependsOn dockerCreateContainer
+    targetContainerId { CONTAINER_NAME }
```

Comment:
Lets change this to: `targetContainerId { dockerCreateContainer.getContainerId() }`.

Lets do this for all the tasks below as well so folks know they can chain things together using the outputs of one task as the inputs to another in true gradle fashion.